### DISCUSSION TITLE
Not Getting Elastic

### DISCUSSION THREAD

**Discussion by Acewiza** [Security-Onion-Solutions/securityonion#10210] [2.4] [answered]
You guys waited until the day after I got Beta-1 running to release Beta-2 just to make me go over the install again right away, didn't you?  ;-)

It went in smooth and looks good - again.  See how much quicker things go when I leave the covers on the machines?

One thing I did notice right away this time tho, is being unable to authenticate anything with an Elastic login.  It tells me correctly when the wrong credentials are entered, but just thinks about it for a second or 2 and flashes back to the correctly filled-in dialog, with no warning banner.

**Comment by dougburks** [comment]
I'm assuming this is the same thing you reported at https://github.com/Security-Onion-Solutions/securityonion/discussions/10501.

Have you tried shift-reload in your browser?

Have you tried clearing your browser cache?

Have you tried a different browser?

**Reply by Acewiza** [reply]
Yes, yes and yes.  I typically use Firefox and Chrome,  Oddly, Elastic authentication DOES work on Chrome, so that appears to be a Firefox issue.

**Reply by dougburks** [reply]
Looks like it works fine for me in Firefox 113.0.2 on an Ubuntu 22.04 VM. But in general, we recommend using Chrome or other chromium-based browsers anyway. From https://docs.securityonion.net/en/2.3/soc.html:
_We recommend chromium or chromium-based browsers such as Google Chrome. Other browsers may work, but fully updated chromium-based browsers provide the best compatibility._

**Reply by Acewiza** [reply, marked as answer]
Now I remember seeing that before.  So we like Chrome - OK.  I go back and forth between FF and Chrome, depending on what's not working or annoying me the most at any given time...
